moml/models/mgnn/djmgnn.py

```
@@ -0,0 +1,424 @@
+                # This assumes edge_attr should have been zeros(0, dim) not None
+                # For safety, if edge_attr is None, new_edge_attr remains None unless super_e is created
+                if edge_attr is None:
+                    new_edge_attr = super_e  # This might be wrong if original edges existed
+                # This part is tricky if edge_attr is None but processed_edge_attr_dim > 0
+                # Let's assume if edge_attr is None, new_edge_attr remains None and GraphConvLayer handles it.
+                # The above cat would fail if edge_attr is None.
+                # So, if edge_attr is None, new_edge_attr should also be None (GraphConvLayer will make dummy)
+                # Or, if we want supernode edges to have *some* attr:
+                if new_edge_attr is None and self.processed_edge_attr_dim > 0:
+                    # Create dummy for all edges if original was None
+                    all_zeros_for_all_edges = x.new_zeros(new_edge_index.size(1), self.processed_edge_attr_dim)
```

Ruff (0.11.9)

Local variable `all_zeros_for_all_edges` is assigned to but never used

Remove assignment to unused variable `all_zeros_for_all_edges`

(F841)